src/app/cli/src/graphql.ml

```diff
@@ -1215,14 +1215,12 @@ module Queries = struct
         let open Result.Let_syntax in
         let propose_public_keys = Coda_lib.propose_public_keys coda in
         let%map pk = Types.Arguments.public_key ~name:"publicKey" pk_string in
-        Option.map
-          Partial_account.(
-            of_pk coda pk |> to_full_account |> Option.map ~f:of_full_account)
-          ~f:(fun account ->
-            { Types.Wallet.account
-            ; is_actively_staking=
-                Public_key.Compressed.Set.mem propose_public_keys pk
-            ; path= Secrets.Wallets.get_path (Coda_lib.wallets coda) pk } ) )
+        (* TODO: return null if the pubkey is not a valid base58check key *)
+        Some
+          { Types.Wallet.account= Partial_account.of_pk coda pk
```

Comment:
ah I think I did the to_full_account round-trip to ensure that this only returned if the public key exists.

See the doc:
```
        "Find any wallet via a public key. Null if the key was not found for \
         some reason (i.e. we're bootstrapping, or the account doesn't exist)"
```

I guess this was surprising behavior on your end, so this should be fine. Can you fix the doc here though?

Reply:
Whoops, didn't see that.
I think it makes sense to be able to query your new wallets here, right?
I was considering doing the full account trick and then if that fails, checking your wallet list to see if it's in there. Would that be better? This seemed simpler for now.